rust-toolchain

```diff
@@ -1,4 +1,2 @@
 [toolchain]
-channel = "nightly-2023-03-15"
```

Comment:
`channel = "1.72.0"`
Why wouldn't use the version the CI uses here?

Reply:
It's some issue with changing Rust toolchain in github CI to stable.
Seems it's only one working solution.